### DISCUSSION TITLE
How do I get a 'browser' object in avalonia browser development？

### DISCUSSION THREAD

**Discussion by zy1075984** [AvaloniaUI/Avalonia#12101] [Q&A]
In R11 avalonia web development, I want to read localStorage information through the browser object, how do I get the 'brower' object?

`var configValue = await browser.ExecuteScriptAsync<string>("window.localStorage.getItem('configKey');");
await browser.ExecuteScriptAsync($"window.localStorage.setItem('configKey', '{configValue}');");`

**Comment by maxkatz6** [comment]
There is none. If you need to interop with browser API or your JS code please see this Microsoft documentation https://learn.microsoft.com/en-us/aspnet/core/client-side/dotnet-interop?view=aspnetcore-7.0
